css/style.css

```diff
@@ -151,4 +163,47 @@
     margin-left: 10px;
 }
 
+.modal-bg {
+    position: fixed;
+    top: 0;
+    left: 0;
+    z-index: 1;
+    overflow: auto;
+    height: 100%;
+    width: 100%;
+    background-color: rgba(170, 170, 170, 0);
+    visibility: hidden;
+}
+
+.modal {
+    position: absolute;
+    width: 850px;
+    height: 830px;
+    top: 8%;
+    left: 28%;
+    background-color: white;
```

Comment:
Previously you used hex code for colors, it is a good pattern to use only one type of coding colors - if you use hex, then use it across application (with exception to rgba)